Documentation~/CustomRulesForRuleTile.md

```diff
@@ -2,13 +2,13 @@
 
 __Contribution by:__ [johnsoncodehk](https://github.com/johnsoncodehk)
 
-This template script is for creating new custom [Rule Tiles](RuleTile.md) with different matching options that differ from the [default options](RuleTile.md#Usage) of ‘Don't Care’, ‘This’ and ‘Not This’. Create clickable options for each Rule in your custom __Rule Tile__.
+Use this template script to create new custom [Rule Tiles](RuleTile.md) with matching options that differ from the Rule Tile’s [default options](RuleTile.md#Usage) (namely **Don't Care**, **This** and **Not This**). This creates selectable options for each Rule in your custom __Rule Tile__.
```

Comment:
**Don't Care** rule is no longer exists.

Reply:
Thanks! Will update documentation for this!

Reply:
Removed Don't Care as a changeable option.